src/index.html

```diff
@@ -22,6 +22,389 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <!-- #region header -->
+    <header class="header">
+      <a
+        href="#"
+        class="logo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="MOYO Logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Photo
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <!-- #endregion header -->
+
+    <!--#region product-cards-->
+
+    <main class="catalog">
+      <div class="product-cards">
+        <div
+          class="product-card"
+          data-qa="card"
+        >
+          <img
+            src="./images/imac.jpeg"
+            alt="imac"
+            class="product-card__image"
+          />
+          <h2 class="product-card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+          <p class="product-card__description">Product code: 195434</p>
+          <div class="mark">
+            <div class="mark__stars">
+              <span class="star"></span>
+              <span class="star"></span>
+              <span class="star"></span>
+              <span class="star"></span>
+              <span class="star"></span>
```

Comment:
You need to use this block the previous tasks, fix classes everywhere and add the modifier class
```suggestion
            <div class="stars stars--4">
              <span class="stars__star"></span>
              <span class="stars__star"></span>
              <span class="stars__star"></span>
              <span class="stars__star"></span>
              <span class="stars__star"></span>
```